monitoring/homepage_test.js

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+/* eslint-disable no-console */
+/* global $http $util $secure */
+/*
+ * Synthetics API Test Documentation:
+ * https://docs.newrelic.com/docs/synthetics/new-relic-synthetics/scripting-monitors/write-synthetics-api-tests
+ */
+const assert = require('assert');
+
+function randomArray(digits, max){
+  return Array.from({length: digits}, () => Math.round(Math.random() * max));
+}
+
+function detectFails(sitePage) {
+  return (response) => {
+    if (response.statusCode !== 200) {
+      assert.fail(`${sitePage} === ${response.statusCode}`);
```

Comment:
Nitpicking but maybe the failure description could be a bit more explicit than just "url === statusCode"

Reply:
Nice! I included the statusMessage ^^ and logged the headers for more details!